src/screens/SurveyOverviewScreen.tsx

```diff
@@ -58,11 +72,18 @@ const renderSectionHeader = ({
   return <SectionHeader title={title} />;
 };
 
-export const SurveyOverviewScreen = () => {
+export const SurveyOverviewScreen = ({ route }: Props) => {
   const { loading, refetch, surveySections } = useSurveySections();
   const RefreshControl = usePullToRefetch(refetch);
   const languages = useSurveyLanguages();
   const navigation = useNavigation<StackNavigationProp<any>>();
+  const { additionalProps } = route.params ?? {};
+
+  const { data } = useStaticContent({
+    name: additionalProps?.htmlName ?? '',
+    type: 'html',
+    refreshTimeKey: 'publicHtmlFile-survey'
```

Comment:
with this hardcoded refresh time key, different `name`s use the same key. having a variable `name` in params to gather different html contents, we should also consider variable refresh time keys. we do this in other places already, have a look around next time please, for example in `HtmlScreen`:

```
const { data, loading, refetch } = useStaticContent({
  name: queryVariables.name,
  type: 'html',
  refreshTimeKey: `${query}-${queryVariables.name}`
});
```

Reply:
Yes, that's right. But since we don't have `query` or `queryVariables` here, I prefer to write hard code. What do you think about that?

```
refreshTimeKey: `publicHtmlFile-${additionalProps?.htmlName}`,
```

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/d3e2bb5ccf4ca1da43e67f69877b7b68207df161

Reply:
yes exactly what i thought of, the variable part is represented